src/main/java/ca/corefacility/bioinformatics/irida/ria/web/cart/dto/CartSample.java

```diff
@@ -0,0 +1,49 @@
+package ca.corefacility.bioinformatics.irida.ria.web.cart.dto;
+
+/**
+ * Used to represent a {@link ca.corefacility.bioinformatics.irida.model.sample.Sample} on the UI Cart Page.
+ */
+public class CartSample {
+	private Long id;
+	private String label;
+	private Project project;
+
+	public CartSample(ca.corefacility.bioinformatics.irida.model.project.Project project, CartSampleRequest sample) {
+		this.id = sample.getId();
+		this.label = sample.getLabel();
+		this.project = new Project(project);
+	}
+
+	public Long getId() {
+		return id;
+	}
+
+	public String getLabel() {
+		return label;
+	}
+
+	public Project getProject() {
+		return project;
+	}
+
+	/**
+	 * The sample needs to know what project it is from.
+	 */
+	class Project {
```

Comment:
The reason why I named it like this it to specifically simplifier the json object returned to the client (which is all this is used for).  Since it is only used as an internal class it should not cause any issue outside the scope of this class.   It will return an object like: 

```json
{
  id: 1,
  label: "Cool Sample Name",
  project: {
    id: 43,
    label: "Awesome Project"
  }
}
```

@tom114 has seen this class.

Reply:
Okay, sounds good.